README.md

```diff
@@ -109,6 +109,6 @@ Get an encrypted value to store in your configuration files.
 
 ## Dependencies
 
-We use async/await so node 8 and up is required.
+Node 6 and up is supported
```

Comment:
Why stop at node v6 here? The transpiled build supports node v4 (tested locally).

Reply:
Mostly to reduce maintenance overhead moving forward. You are right that it should be fine currently but I would like to develop some additional features in this library moving forward and retaining backwards compatibility can be a bit of a pain.

Additionally, from a more philosophical perspective I don't want to be supporting an end of life version that won't be receiving security updates etc (which will be the case at the end of April).

Hope that makes sense!

Reply:
makes perfect sense, thanks for the updates!